server/server.ts

```diff
@@ -152,7 +157,7 @@ app.listen(PORT, () => {
   console.clear();
   printLogo();
   // TODO: Add version/build number here
-  console.log('Listening on ', colors.grey(`http://localhost:${PORT}`));
+  logger.debug(`Listening on ${colors.gray(`http://localhost:${PORT}`)}`);
```

Comment:
maybe try to implement it in a way that `debug` logs will always log in `grey`, `error` always in `red` and so on...

Reply:
We use it only here to put the listening address in color, the rest will not use this.
With `log4js` every log level have different color for the stimestamp and tag while the log itself is white.

Reply:
👍